crates/cargo-contract/src/cmd/info.rs

```diff
@@ -55,6 +61,9 @@ pub struct InfoCommand {
     /// Export the instantiate output in JSON format.
     #[clap(name = "output-json", long)]
     output_json: bool,
+    /// Display the pristine Wasm code.
```

Comment:
```suggestion
    /// Display the contract's Wasm bytecode.
```

Reply:
The context for this change is that "pristine" is a term that is used only in `pallet-contracts` internally. So for user-facing output of `cargo contract` we should just use `wasm` instead of `pristine_wasm`.